modules/margin-protocol/src/lib.rs

```diff
@@ -100,6 +100,12 @@ decl_event! {
 		Deposited(AccountId, Amount),
 		/// Withdrew: (who, amount)
 		Withdrew(AccountId, Amount),
+		/// TraderMarginCall: (who)
+		TraderMarginCall(AccountId),
```

Comment:
```suggestion
		TraderMarginCalled(AccountId),
```

Use past tense for events, same apply to the other two events.